docs/debug.md

```diff
@@ -161,16 +160,16 @@ Look for  `failed to read the file` or `unexpected copied data size for on-deman
 
 **Network Failures**
 
-The snapshotter contains custom retry logic when fetching spans(data) from the remote registry. By default it will try to fetch from remote a maximum of 5 times before returning an error.
+The snapshotter contains custom retry logic when fetching spans(data) from the remote registry. By default it will try to fetch from remote a maximum of 9 times before returning an error.
 
-* You can look for `Retrying request` within the logs to determine the error and response returned from the remote registry.
+* You can look for `retrying request` within the logs to determine the error and response returned from the remote registry.
 * You can also check `operation_duration_remote_registry_get` metric to see how long it takes to complete `GET` from remote registry.
 
 # Debugging Tools
 
 ## CLI
 
-Here are some SOCI CLI commands that can be helpful when debugging issues relating to SOCI indices/zTOCs'
+Here are some `soci` CLI commands that can be helpful when debugging issues relating to SOCI indices/zTOCs'
```

Comment:
I think the usage was correct before. We should say either "Here are some SOCI CLI commands" or "Here are some `soci` commands". Since the project is called SOCI, I think "SOCI CLI" does better to disambiguate, but either one would be ok.

As a point of reference, we say `soci-snapshotter-grpc` or SOCI snapshotter, not `soci-snapshotter-grpc` snapshotter.